tests/Rules/Deprecations/CallToDeprecatedStaticMethodRuleTest.php

```diff
@@ -57,10 +61,6 @@ public function testDeprecatedStaticMethodCall(): void
 					"Call to deprecated method deprecatedWithDescription() of class CheckDeprecatedStaticMethodCall\Foo:\nThis is probably a singleton.",
 					16,
 				],
-				[
-					'Call to deprecated method deprecatedFoo() of class CheckDeprecatedStaticMethodCall\Foo.',
-					24,
```

Comment:
Thank you! One more thing I need here is to actually add a test that checks that this scenario is continuted to be reported in a valid scenario, e.g. calling `parent::deprecatedFoo();` in a scope that's not deprecated. Thanks.

Reply:
Hm ok. I will see if I can get that fixed. If not I will let you know

Reply:
If I understood your post correctly, I think this has been added back in another way now?

Reply:
No, it's just that this extension is built with latest stable PHPStan version and not with dev-master. Update the composer.json for the purpose of this PR :)

Reply:
Yes, I know that. I was referring to the actual meaning of your sentence 🤓

Anyway updates composer.json as well now